### ISSUE TITLE
Doesn't handle Meta well

### ISSUE THREAD

**Issue by dan-passaro** [landscapeio/pylint-django#50]
I have this `Meta` class in one of my models:

```
class Meta:
    unique_together = ('obj_1_id', 'obj_2_id', 'content_type')
```

This seemingly fine `Meta` class generates these errors using pylint and pylint-django:

```
$ pylint --rcfile /dev/null -r n --load-plugins pylint_django models.py
[...]
C: 40, 8: Missing class docstring (missing-docstring)
W: 40, 8: Class has no __init__ method (no-init)
C: 40, 8: Old-style class defined. (old-style-class)
R: 40, 8: Too few public methods (0/2) (too-few-public-methods)
W: 40, 8: Unused variable 'Meta' (unused-variable)
```

I have these versions:

```
pylint==1.4.3
pylint-django==0.6.1
pylint-plugin-utils==0.2.3
```

**Comment by vinnking** [comment]
Please refer to the article: https://blog.landscape.io/using-pylint-on-django-projects-with-pylint-django.html

**Comment by bittner** [comment]
I have the same problem: `missing-docstring` and `too-few-public-methods` is reported for the `Meta` class (of a Form subclass) in my case.

Interestingly, this _is_ being handled by this plugin [according to the source code](https://github.com/landscapeio/pylint-django/blob/master/pylint_django/augmentations/__init__.py#L727-L732). Not sure what is the problem here. Software versions I'm using:
```
Python==3.5.2
Django==1.10.4
astroid==1.4.9
pylint==1.6.4
pylint-django==0.7.2
tox==2.5.0
```
Is this maybe a problem with the Python and/or Django version I'm using? (Because I also have issues with `Invalid constant name "urlpatterns"`, see issue #41).

**Comment by atodorov** [comment]
Can't reproduce with the latest updates so closing. If you can reproduce open another issue with the code to reproduce.
